### PR TITLE
CR017: add GTFS-R enums to Occupancy

### PR DESCRIPTION
**Pull request by haeckerbaer**
Change Request for the addition of the GTFS-R occupancy values to the SIRI Occupancy Enumeration. The GTFS values are added after the existing SIRI values. See documentation in the basecamp:
[Siri_CR_017_Harmonisation_Occupancy.doc](https://3.basecamp.com/3256016/buckets/9672657/uploads/1696454060)

### CODE REVIEW

File: xsd/siri_model/siri_reference-v2.0.xsd [hunk 1]
```diff
@@ -632,6 +632,37 @@ Values for these elements can be specified on an annual schedule and will be inh
...
+    <xsd:annotation>
+     <xsd:documentation>The vehicle has a small percentage of seats available. What percentage of free seats out of the total seats available is to be considered small enough to fall into this category is determined at the discretion of the producer.</xsd:documentation>
+    </xsd:annotation>
+   </xsd:enumeration>
+   <xsd:enumeration value="standingRoomOnly">
+    <xsd:annotation>
+     <xsd:documentation>The vehicle can currently accommodate only standing passengers.</xsd:documentation>
+    </xsd:annotation>
+   </xsd:enumeration>
+   <xsd:enumeration value="CrushedStandingRoomOnly">
```
Comment (skinkie) [inline xsd/siri_model/siri_reference-v2.0.xsd]: I would suggest lower crushedStandingRoomOnly.

File: xsd/siri_model/siri_reference-v2.0.xsd [hunk 2]
```diff
@@ -632,6 +632,37 @@ Values for these elements can be specified on an annual schedule and will be inh
...
+    <xsd:annotation>
+     <xsd:documentation>The vehicle can currently accommodate only standing passengers and has limited space for them.</xsd:documentation>
+    </xsd:annotation>
+   </xsd:enumeration>
+   <xsd:enumeration value="NotAcceptingPassengers">
```
Comment (skinkie) [inline xsd/siri_model/siri_reference-v2.0.xsd]: I would suggest lower notAcceptingPassengers

File: xsd/siri_model/siri_reference-v2.0.xsd (hunk 1 above)
Comment (haeckerbaer) [reply xsd/siri_model/siri_reference-v2.0.xsd]: @skinkie Thanks for your input! I corrected it with the new commit.

File: xsd/siri_model/siri_reference-v2.0.xsd (hunk 2 above)
Comment (haeckerbaer) [reply xsd/siri_model/siri_reference-v2.0.xsd]: @skinkie Thanks for your input! I corrected it with the new commit.

**Pull request merged**